From be3127f7cb882403812c0154050f567aabfb60e7 Mon Sep 17 00:00:00 2001
From: Quintin Hill <Quintin.Hill@arqiva.com>
Date: Thu, 13 Apr 2017 12:13:26 +0100
Subject: [PATCH 3/6] condor_snutils: Visual Studio 2015+ has correct snprintf

Fix build on Visual Studio 2015+ which now supplies a C99 compliant
snprintf and vsnprintf in the standard library.
---
 src/condor_includes/condor_snutils.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/condor_includes/condor_snutils.h b/src/condor_includes/condor_snutils.h
index 6a2c8708e5..071f173ff2 100644
--- a/src/condor_includes/condor_snutils.h
+++ b/src/condor_includes/condor_snutils.h
@@ -40,6 +40,7 @@
 BEGIN_C_DECLS
 
 #ifdef WIN32
+#if _MSC_VER < 1900
 int snprintf(char *str, size_t size, const char *format, ...);
 /**	Disable the warning about the number of formal parameters 
 	differing from a previous declaration */
@@ -47,6 +48,7 @@ int snprintf(char *str, size_t size, const char *format, ...);
 #pragma warning(suppress: 28251) // inconsistent annotations
 int __cdecl vsnprintf(char *str, size_t size, const char *format, va_list args);
 #endif
+#endif
 
 int printf_length(const char *format, ...);
 int vprintf_length(const char *format, va_list args);
-- 
2.13.5

